scripts/testnet/upgrade.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -eu 
+
+UPGRADE_NAME="$1"
+UPGRADE_COMMIT_HASH="$2"
+
+COSMOVISOR_HOME=/stride/.stride/cosmovisor
+
+echo "Upgrade Name: $UPGRADE_NAME"
+printf "Upgrade Commit Hash: $UPGRADE_COMMIT_HASH\n\n"
+
+while true; do
+    read -p "Continue? [y/n]" yn
+    case $yn in
+        [Yy]* ) echo ""; break;;
+        [Nn]* ) exit ;;
+        * ) printf "Please answer yes or no.\n";;
+    esac
+done
+
+UPGRADE_DIR=${COSMOVISOR_HOME}/upgrades/${UPGRADE_NAME}/bin
+
+mkdir -p $UPGRADE_DIR
```

Comment:
does this script assume the upgrade is being run from a particular directory?

Reply:
Yeah this is run from inside the docker image. Its sitting in the home directory after you exec into the container